app/utils/roles.js

```diff
@@ -13,7 +15,7 @@ module.exports = {
   getRoleByExtId: roleExtId => {
     let found
     _.toArray(roles).forEach(role => {
-      if (role.extId === roleExtId) {
+      if (role.extId === roleExtId && role.extId !== 500 && role.extId !== 600) {
```

Comment:
I don't think this is a good precedent to set, it's not clear from the code what 500 or 600 are or why they'd be excluded. 

If this pattern had to exist I would recommend naming this and separating the inline logic so that it's explicit, something along the lines of

```js 
const nonPermissableRoleExternalIds = [ 500, 600 ]
if (nonPermissableRoleExternalIds.includes(role.extId)) {
  return
}
```

This whole function and patten is confusing to be honest

Reply:
This will be gone soon, hopefully in a few hours.